src/Sylius/Bundle/CoreBundle/Migrations/Version20201204071301.php

```diff
@@ -30,7 +30,7 @@ public function up(Schema $schema): void
             return;
         }
 
-        $this->addSql('ALTER TABLE sylius_adjustment ADD details JSON NOT NULL');
+        $this->addSql('ALTER TABLE sylius_adjustment ADD details LONGTEXT NOT NULL');
```

Comment:
```suggestion
        $this->addSql('ALTER TABLE sylius_adjustment ADD details LONGTEXT NOT NULL COMMENT \'(DC2Type:json)\'');
```

Include the Doctrine type comment so values can be reverse engineered correctly.

Reply:
Thank you. 👍